src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -2294,7 +2294,7 @@ private void SetBackgroundColor(Color value)
 
             if (backgroundExtraData == null)
             {
-                Object.InternalSetPropertyVector4(SwigCPtr, View.Property.BACKGROUND, ((Color)value).SwigCPtr);
+                Object.InternalSetPropertyVector4(SwigCPtr, View.Property.BACKGROUND, value.SwigCPtr);
```

Comment:
I don't know, so I am just asking...
here, is this code needed?
`value.CopyValuesFromDali();`

Reply:
No, because when try to get the SwigCPtr of Color,